packages/web-components/package.json

```diff
@@ -34,8 +34,10 @@
     "prettier": "prettier --config ../../prettier.config.js --write 'src/**/(*.ts|*.html)' --ignore-path ../../.prettierignore",
     "prettier:diff": "prettier --config ../../prettier.config.js 'src/**/(*.ts|*.html)' --ignore-path ../../.prettierignore --list-different",
     "code-style": "npm run prettier && npm run lint",
-    "start": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
+    "start": "yarn start-storybook -p 6006",
+    "start-storybook": "../../node_modules/@storybook/html/bin/index.js",
```

Comment:
💪 nice. 

in future this will be mitigated/abstracted away by nx storybook executor https://github.com/nrwl/nx/blob/master/packages/storybook/src/executors/storybook/schema.json#L7